chall01/ksappi.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+import sys
+
+charset = [".-", "-...", "-.-.", "-..", ".", "..-.", "--.", "....", "..",
+	".---", "-.-", ".-..", "--", "-.", "---", ".--.", "--.-", ".-.",
+	"...", "-", "..-", "...-", ".--", "-..-", "-.--", "--.."]
+
+def main():
+	output = ""
```

Comment:
small note:
just a good practice to initialize a variable you'll use in the loop right before the loop and not in the begging of function
especially if this variable will only be modified inside a loop, so it's kind of a "loop" variable, and if it's initialized somewhere else it's hard to track